docker-compose.yml

```
@@ -1,9 +1,19 @@
 version: '3.7'
 
 services:
-  backend:
-    container_name: backend
-    image: backend
-    command: npm run start:dev
+  nidos-backend:
+    container_name: nidos-backend
+    image: nidos-backend
+    command: yarn run start:dev
```

```suggestion
    command: yarn start
```

if this should run the prod version?